src/trade_executor.h

```diff
@@ -51,12 +51,25 @@ class TradeExecutor {
 
   /// @brief execute all trades, collecting responders from bidders and sending
   /// responses to requesters
+  /// @deprecated Use ExecuteTrades(Context*) instead.
   void ExecuteTrades() {
+    Warn<IO_WARNING>("this function does not record trades to the database");
```

Comment:
@rwcarlsen, it looks like this function is just implementing what is now ExecuteTrades(NULL). Can you update this to reflect that?